docs/wiki/NuGet-packages.md

```diff
@@ -13,6 +13,40 @@ This package is primarily intended as a method for rapidly shipping hotfixes to
 Note: this package is intended as a replacement for Microsoft.Net.Compilers (which is a Windows-only package) and Microsoft.NETCore.Compilers. Those packages are now deprecated and will be deleted in the future.
 
 ## Versioning
+Starting on Roslyn versions `5.0` and above, Roslyn NuGet packages are shipped with the .NET SDK instead of VS.  New Roslyn packages will be published on every SDK preview and GA release.
```

Comment:
In practice we've had challenges when we consume new VS/editor packages, since those sometimes aren't available until a VS release, and those can take a few days to get posted. Does moving this to SDK releases make anything more challenging there?

Reply:
Yes and no.  We still need to publish manually because we may have to update for new VS dependencies.  If VS releases don't align with the SDK and release slower the SDK there is the possibility of us having to wait for a VS release to publish packages.  Can happen today for prereleases for new major versions